tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetBase.cs

```diff
@@ -53,6 +53,12 @@ public AspNetBase(string sampleName, ITestOutputHelper outputHelper, string shut
             // adding these header so we can later assert it was collected properly
             _httpClient.DefaultRequestHeaders.Add(XffHeader, MainIp);
             _httpClient.DefaultRequestHeaders.Add("user-agent", "Mistake Not...");
+
+#if NETCOREAPP2_1
```

Comment:
You've done this already somewhere else IIRC? Are there any other aspnet tests we should modify as well to avoid doing it iteratively?